scopes/design/ui/Avatar/user-avatar.tsx

```diff
@@ -1,19 +1,37 @@
 import React, { PureComponent } from 'react';
 import classNames from 'classnames';
+import Tooltip, { Place, Type, Effect } from 'react-tooltip';
+import { v1 } from 'uuid';
 import { Icon } from '@teambit/evangelist.elements.icon';
 import { addAvatarQueryParams } from '@teambit/url.add-avatar-query-params';
 import { getInitials } from '@teambit/string.get-initials';
 import { letterBgColors } from '@teambit/ui.styles.colors-by-letter';
+import { ellipsis } from '@teambit/ui.styles.ellipsis';
 import { AccountObj } from './avatar';
 import styles from './styles.module.scss';
 
+type tooltipProps = {
+  place: Place;
+  type: Type;
+  effect: Effect;
+};
+const tooltipDefaultProps: tooltipProps = {
+  place: 'bottom',
+  type: 'dark',
+  effect: 'solid',
+};
+
 export type UserAvatarProps = {
   account: AccountObj;
   size: number;
   imageSize?: number;
   fontSize?: number;
   className?: string;
   imgClassName?: string;
+  /**
+   * showing or not a tooltip when hover on the avatar, this value is true by default
+   */
+  hideTooltip?: boolean;
```

Comment:
don't use doubleNegative! instead of
```
<Tooltip hideTooltip={false}  {...}  />
```

it's better to have 
```
<Tooltip showTooltip {...} />
```

or, if the **default is to show tooltip**, you can use hideTooltip as without `={true}`
```
<Tooltip hideTooltip  {...}  />
```